src/peft/tuners/adaption_prompt/layer.py

```diff
@@ -75,21 +75,25 @@ def forward(self, **kwargs):
         k_proj_layer = TRANSFORMERS_MODEL_CONFIG[self.model_type].k_proj_layer
         v_proj_layer = TRANSFORMERS_MODEL_CONFIG[self.model_type].v_proj_layer
         o_proj_layer = TRANSFORMERS_MODEL_CONFIG[self.model_type].o_proj_layer
+        factor = (
```

Comment:
I wonder if it would be more cautious to check that the two values divide without remainder. Or should it be impossible to have a remainder?

factor, remainder = divmod(self.model.k_proj.in_features // self.model.k_proj.out_features)
if remainder:
    raise ValueError(...)

Reply:
Thank you for the prompt reply!
Upon checking [modeling_mistral.py](https://github.com/huggingface/transformers/blob/1c31b7aa3bb4e7ef24c77596d2a76f45a770159f/src/transformers/models/mistral/modeling_mistral.py#L214), it seems like it should not be possible to have a remainder. 
Please let me know your thoughts.